runconfig/merge.go

```diff
@@ -33,6 +33,7 @@ func Merge(userConf, imageConf *Config) error {
 				userEnvKey := strings.Split(userEnv, "=")[0]
 				if imageEnvKey == userEnvKey {
 					found = true
+					break
```

Comment:
just remove `if !found`, if you reached that code, then you're not breaked - `!found`